db/schema.rb

```diff
@@ -1439,9 +1439,9 @@
 
   create_table "users_gdpr_delete_requests", force: :cascade do |t|
     t.datetime "created_at", precision: 6, null: false
-    t.string "email"
+    t.string "email", null: false
```

Comment:
This seems to have snuck in, there's no migration that would cause this.

Reply:
gdi, good catch 🤦🏻‍♀️